documentation/about/sc-tor.md

```diff
@@ -32,33 +37,31 @@
+   5.1 The Committee will meet at least two but up to four times a year to review the 360Giving Data Standard, formats and classifications and to discuss whether any changes or upgrades are required.
 
-   5.2 The Committee is expected to meet in person at least once per annum.
+   5.2 The Chair shall convene and preside over all Committee meetings, coordinating with the Chief Executive of 360Giving or Product Manager as required to ensure that members receive relevant documentation in advance. Additional meetings may be convened at the request of two Committee members. In the Chair’s absence, another member can be nominated by those present to preside.
 
-   5.3 The Chair shall convene and preside over all Committee meetings, coordinating with the Chief Executive of 360Giving as required to ensure that members receive relevant documentation in advance. Additional meetings may be convened at the request of two Committee members. In the Chair’s absence, another member can be nominated by those present to preside.
+   5.3 At all times the Committee will work to promote, support and uphold the principles of transparency in its activities by making full and accurate information about its activities and proposals publicly available so that anyone can review them.
 
-   5.4 At all times the Committee will work to promote, support and uphold the principles of transparency in its activities by making full and accurate information about its activities and proposals publicly available so that anyone can review them.
-
-These Terms of Reference were prepared on 16th August 2016 by 360Giving and approved by members at the inaugural Stewardship Committee meeting on 1st December 2016.
+   5.4 360Giving and the Committee share a commitment to valuing diversity, equity and inclusion. At all times the Committee will operate in line with the 360Giving Diversity, Equity and Inclusion Policy and action plan, and the 360Giving core values.
 
 ***6\. Conflicts of interest***
 
    6.1 Committee members are to declare any direct or indirect interest (whether personal, or by a duty of loyalty to another organisation) that conflicts, or might conflict with the interests of 360Giving and the appropriate maintenance of the Standard at the outset of any discussion on the relevant issue.
 
    6.2 Any declaration of a conflict of interest is to be noted in the public record of discussions, together with details of any proposals made.
 
-***7\. Individual member’s responsibilities*** 
+***7\. Individual members' responsibilities*** 
 
    7.1 Committee members will at all times:
 
 - Remain appropriately informed on 360Giving’s open data Standard and operating environment.
 - Endeavour to attend all Committee meetings, ensuring that they are adequately prepared to contribute to the discussion.
-- Exercise independence of judgment, acting legally and in good faith to promote and protect 360Giving’s interests.
```

Comment:
- Exercise independence of judgment, acting legally and in good faith to promote and protect 360Giving’s interests.